index.html

```diff
@@ -149,9 +135,63 @@
                         <p>Work toward your own service enterprise.</p>            
                     </div>
                     
-                </section> <!-- end myIconRow -->
+                </section> <!-- end outside-in-iconRow -->
 
 
+                <section class="row outside-in-infoRow1 outside-in-rowPadding">
+                
+                    <div class="small-12 column">
+                        <h3>For organizations that need help with accessibility</h3>
+                    </div>
+                    
+                    <div class="small-12 medium-6 column">
+                        <p>Outside-In takes your inaccessible documents and audiovisual media and makes them accessible. We use your documents to provide training and work to young people with episodic disabilities and provide you with a quality-assured accessible file. We also provide website accessibility reports and can help you understand any changes you should make to your website to make it accessible and compliant with legislation such as the Accessibility for Ontarians with Disabilities Act. If you would like to work with Outside-In to meet your accessible media needs, please contact the project coordinator to discuss your accessibility needs and rates.</p>
+                    </div>
+                    
+                    <div class="small-12 medium-6 column outside-in-infoSideColumn">
+                    
+                        <h4>Our services</h4> 
+                        
+                        <ul>
+                            <li>PDF document remediation </li>
+                            <li>Word document remediation</li>
+                            <li>Video captioning</li>
+                            <li>Website accessibility reports</li>
+                        </ul>
+                        
+                    </div>
+                                      
+                </section> <!-- end outside-in-infoRow1 -->
+                
+                <section class="row outside-in-infoRow2 outside-in-rowPadding">
+                
+                    <div class="small-12 column">
+                        <h3>For youth with episodic disabilities</h3>
+                    </div>
+                    
+                    <div class="small-12 medium-6 column">
+                        <p>Outside-In provides free online training for young people (ages 13-29) who know or think that they have an invisible and/or episodic disability that makes it challenging to gain formal training or maintain traditional work hours. This training will teach participants how to make digital media more accessible, and includes topics such as: making digital office documents such as Word and PDF accessible, adding closed captioning to videos, and reviewing websites for accessibility. Participants can learn at their own pace and will be under the guidance of a mentor.</p>
```

Comment:
I had exactly the same thought - but decided to leave it - not sure if we want the whole thing to be in second-person ('you') or not. What I did do was at least make each section consistent (it was more mixed before)...